packages/expo-random/src/__tests__/Random-test.ts

```diff
@@ -0,0 +1,15 @@
+import { mockProperty, unmockProperty } from 'jest-expo';
+
+import ExpoRandom from '../ExpoRandom';
+import * as Random from '../Random';
+
+it(`invokes native method correctly`, async () => {
+  mockProperty(ExpoRandom, 'getRandomIntegerAsync', jest.fn(async () => ''));
+
+  const value = await Random.getRandomIntegerAsync(52);
+
+  expect(value instanceof Uint8Array).toBeTruthy();
+  expect(ExpoRandom.getRandomIntegerAsync).toHaveBeenLastCalledWith(52);
+
+  unmockProperty(ExpoRandom, 'getRandomIntegerAsync');
+});
```

Comment:
Add a test where the native method mock returns an actual Base64 with the expected number of random bytes.

```
it(`returns an array with the desired number of bytes`, async () => {
  ExpoRandom.getRandomIntegerAsync.mockImplementation(async () => some string that actually has random data of the right length);
  const value = await Random.getRandomIntegerAsync(8);
  expect(value.length).toBe(8);
});